Makefile

```diff
@@ -253,6 +253,9 @@ all: addon server
 .PHONY: clean
 clean:
 	rm -rf build/ addon/webextension/build/ addon/webextension/manifest.json addon/webextension/_locales/
+
+.PHONY: distclean
+distclean: clean
```

Comment:
Maybe this should also rm -rf node_modules/ ?  But I'm not entirely sure what distclean should do compared to clean